### ISSUE TITLE
Tuple/list split across lines gets rewritten to invalid syntax

### ISSUE THREAD

**Issue by chriskuehl** [asottile/add-trailing-comma#29]
```
x = ([a,
     b], None)
```

gets rewritten to this:

```
x = (     
    [     
        a,
        b,
    ],    
, None    
)         
```

which is invalid syntax:
```
  File "test.py", line 6   
    , None                 
    ^                      
SyntaxError: invalid syntax
```

(with version 0.6.0)

**Comment by asottile** [comment]
Very strange, bisects to https://github.com/asottile/add-trailing-comma/commit/48b79ddd75ef64618a6ad0b68988382c4d52d56c

Time to debug this one! Stupid tuples!

**Comment by asottile** [comment]
Fix is in 0.6.1!
